<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateFS</class>
 <widget class="QDialog" name="CreateFS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create file system</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="pathLabel">
        <property name="text">
         <string>Path:</string>
        </property>
        <property name="buddy">
         <cstring>pathText</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="pathLayout">
        <item>
         <widget class="QLineEdit" name="pathText"/>
        </item>
        <item>
         <widget class="QPushButton" name="pathButton">
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fileTypeLabel">
        <property name="text">
         <string>File system type:</string>
        </property>
        <property name="buddy">
         <cstring>fileTypeBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="fileTypeBox">
        <item>
         <property name="text">
          <string>MINIX v3</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fileSizeLabel">
        <property name="text">
         <string>Size:</string>
        </property>
        <property name="buddy">
         <cstring>fileSizeBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="fileSizeBox">
        <property name="suffix">
         <string> MiB</string>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
